protocol/src/device/probe.rs

```diff
@@ -74,6 +76,10 @@ pub fn probe_kind(device_info: &DeviceInfo) -> DeviceKind {
         (2, 22) => M2x4,
         #[cfg(feature = "device_flex")]
         (27, 100) => Flex,
+
+        #[cfg(feature = "device_flex_ht")]
+        (32, 113) => Flex, //get from `minidsp probe`
```

Comment:
The HTx appears to share the same device id. I cannot think of a brief way to make that evident in the device name itself - however, would you mind adding a comment here (or somewhere) that  `device_flex_ht` also supports Flex HTx?

Reply:
Great point, I actually have the HTx not the HT.  I've updated the name to reflect